From a7e5e7324e4f4b3a60b494638abd0154b3828c00 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Wed, 19 Nov 2014 23:04:32 +0100
Subject: llvm: don't use --system-libs

this continues the saga of not linking the .a files so we don't care
about the libs llvm links against (like -ledit)

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 libclamav/c++/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: clamav-0.98.7+dfsg/libclamav/c++/configure.ac
===================================================================
--- clamav-0.98.7+dfsg.orig/libclamav/c++/configure.ac	2015-05-04 11:36:44.652384159 -0400
+++ clamav-0.98.7+dfsg/libclamav/c++/configure.ac	2015-05-04 11:37:10.852662874 -0400
@@ -141,7 +141,7 @@
         if test $llvmver_test -ge 350; then
            dnl LLVM 3.5.0 and after splits linker flags into two sets
            ldflags=`$llvmconfig --ldflags`
-           syslibs=`$llvmconfig --system-libs`
+           syslibs=
            AC_SUBST(LLVMCONFIG_LDFLAGS, ["$ldflags $syslibs"])
         else
            AC_SUBST(LLVMCONFIG_LDFLAGS, [`$llvmconfig --ldflags`])
